### PR TITLE
TS - Hackpacks

### LINKED ISSUE #1638: TypeScript: Hackpacks
image

### PR DESCRIPTION
**Pull request by kathalinne**

### CODE REVIEW

File: src/data/user/Hackpacks.ts [hunk 1]
```diff
@@ -7,18 +7,31 @@ import {
   SiCplusplus as CPlusPlus,
   SiMongodb as Mongodb,
 } from "@icons-pack/react-simple-icons";
-import { ReactNode } from "react";
+// import { ReactNode } from "react";
```
Comment (shahdivyank) [inline src/data/user/Hackpacks.ts]: remove comment, you can delete the line

File: src/components/user/resources/Hackpack.tsx [hunk 2]
```diff
@@ -25,18 +31,22 @@ const Hackpack = ({ text, techs, link, description }: props) => {
...
+          const IconComponent = getIconComponent(tech);
+          return (
+            <div
+              key={index}
+              className="flex items-center gap-2 text-gray-400"
+              data-cy="hackpack-tech"
+            >
+              <div className="text-hackathon-blue-100" data-cy="hackpack-icon">
+                {IconComponent ? <IconComponent /> : null}{" "}
+                {/* Render the icon component */}
```
Comment (shahdivyank) [inline src/components/user/resources/Hackpack.tsx]: delete the comment

**Comment by kathalinne** [comment]
fixed

**Comment by shahdivyank** [comment]
> fixed

put a question in discord as well

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**